crates/db_schema/src/impls/mod_log/admin.rs

```diff
@@ -19,25 +39,23 @@ use diesel_async::RunQueryDsl;
 impl Crud for AdminPurgePerson {
   type InsertForm = AdminPurgePersonForm;
   type UpdateForm = AdminPurgePersonForm;
-  type IdType = i32;
+  type IdType = AdminPurgePersonId;
```

Comment:
The changes in all these `db_schema` files are just adding the type, and some cleaner imports. So no need to read through them all.